.github/workflows/bump-version-and-update-milestone.yml

```diff
@@ -38,4 +38,4 @@ jobs:
         run: |
           .ci/update-github-milestone.sh ${{ github.event.inputs.version }}
         env:
-          READ_ONLY_TOKEN: ${{ github.token }}
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
Please do me favor and move env block above run.

Reply:
Done. Changed the commit message as suggested in issue description as well.